pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -1,56 +1,42 @@
-// This file and its contents are licensed under the Apache License 2.0.
-// Please see the included NOTICE for copyright information and
-// LICENSE for a copy of the license.
-
-package pgmodel
+package ingestor
```

Comment:
Missing copyright header.

Reply:
BTW, do we need to have the copyright header across each and every file in the project? (Just for my info)